src/app/components/toolbar/toolbar-guidelines.component.html

```diff
@@ -3,14 +3,8 @@
   <p>
     When a content container requires actions, place them in a toolbar. Only use toolbars in <a routerLink="/components/list">lists</a>, below the <a routerLink="/components/page-summary">page summaries</a> on record pages, or in <a routerLink="/components/tiles">tiles</a>. Toolbars are not appropriate in any other situations.
   </p>
-  <div class="screenshot screenshot-info">
-    <img src="~/assets/img/guidelines/toolbar/generic-toolbar.png" style="width: 100%;" />
-  </div>
-</section>
 
-<section>
-  <header><div class="sky-section-heading">Placement</div></header>
-  <div class="sky-subsection-heading">List toolbar</div>
+  <div class="sky-subsection-heading"><a routerLink="/components/list/toolbar">List toolbar</a></div>
   <stache-row>
     <stache-column screenSmall="6">
```

Comment:
I know the text in the List toolbar section was already in place on the toolbar component docs, but reading through it again, "It contains actions to manipulate the contents and display of the list" reads awkwardly. At least the "display of the list" part does. I'd change "display" to "appearance" unless we mean something else by "display" here.

Reply:
see comment above re "display" vs "appearance"